css-two-column-layout/styles.css

```diff
@@ -4,7 +4,7 @@ body {
 }
 
 .container {
-  margin: 5%;
+  max-width: fit-content;
```

Comment:
This will end up just being normal `block` behavior. I would look back to `css-layout-classes` for a reminder of the right sort of configuration of a `container` layout class to get the desired behavior of limiting the width of the content and centering the content in the remaining space.